server/channels/db/migrations/migrations.list

```diff
@@ -218,8 +218,12 @@ channels/db/migrations/mysql/000108_remove_orphaned_oauth_preferences.down.sql
 channels/db/migrations/mysql/000108_remove_orphaned_oauth_preferences.up.sql
 channels/db/migrations/mysql/000109_create_persistent_notifications.down.sql
 channels/db/migrations/mysql/000109_create_persistent_notifications.up.sql
+channels/db/migrations/mysql/000110_create_desktop_tokens.down.sql
+channels/db/migrations/mysql/000110_create_desktop_tokens.up.sql
```

Comment:
I see that you are using 112 separately. There is no need to add 110 anymore. The migration from 110 has already been deployed and exists on cloud.

Reply:
Just adding it back since this PR (including the migration) was reverted. So if it's not added here it won't be in the codebase at all.

Reply:
Right, that's fine.